pybind11/how-to.rst

```diff
@@ -461,6 +467,39 @@ The end result of all the steps above looks like this:
 
     }}  // lsst::tmpl
 
+Building the wrapper
+--------------------
+
+The next step is to tell SCons to build your wrapper.
+Edit ``python/.../SConscript`` to look like this:
+
+.. code-block:: python
+
+    # -*- python -*-
+    from lsst.sconsUtils import scripts
+    scripts.BasicSConscript.pybind11(
+        [
+            'exampleOne',
+            # ... list additional pybind11 wrappers, if any
+        ],
+        addUnderscore=False,
+    )
+
+``addUnderscore`` is historical baggage; eventually we plan to make ``False`` the default.
+
+Importing the wrapper
+---------------------
+
+The Python name for your wrapper module is `exampleOne`.
+If the wrapped classes can be returned by a function or unpickled then it is crucial that your module is imported when the package is imported.
+If the symbols are part of the public API then this is typically done by adding the following to your package's main ``__init__.py`` file:
+
+.. code-block:: python
+
+    from exampleOne import *
+
+If you don't want your wrapper's symbols in your package's top-level namespace then you can use ``from . import exampleOne`` or import the module in some other way.
```

Comment:
"or import the module in some other way" - I have no idea what the other options would be.

Reply:
Following "explicit is better than implicit", I'd suggest stopping the sentence at `exampleOne`. I think we're much better off having two explicit options (`from X import *` and `from . import X`), and not implying the others, which are too implicit and "magic" to me.